recipe/meta.yaml

```diff
@@ -9,19 +9,22 @@ source:
   sha256: 7f6b2733bde06ea2575b79a6450d9bd5c38918ff4cb2a355bf22bbe8c86c6bcf
 
 build:
+  # skip py<39 required from upstream
```

Comment:
Yeah, comment can be removed https://github.com/h5netcdf/h5netcdf/blob/2951cf8ea3174dde5663451484bde0cebaa535d0/pyproject.toml#L12
```suggestion
```

Reply:
Ok, so the comment is not necessary, but can you confirm that the selector IS necessary because of the upstream requirements?